RadixWallet/Features/AccountRecoveryScan/Coordinator/AccountRecoveryScanCoordinator.swift

```diff
@@ -85,20 +74,23 @@ public struct AccountRecoveryScanCoordinator: Sendable, FeatureReducer {
 		case dismissed
 	}
 
+	@Dependency(\.secureStorageClient) var secureStorageClient
 	@Dependency(\.onboardingClient) var onboardingClient
 	@Dependency(\.accountsClient) var accountsClient
 	@Dependency(\.dismiss) var dismiss
 	public init() {}
 
 	public var body: some ReducerOf<Self> {
-		Scope(state: \.root, action: /Action.child .. ChildAction.root) {
-			AccountRecoveryScanInProgress()
+		Scope(state: \.root, action: /Action.child) {
+			EmptyReducer()
```

Comment:
@kugel3 hmm not exactly the same, since `DappInteractor` has
```swift
@PresentationState
var currentModal: Modal.State?
```
whereas I have `root: Root`. Hmm I dont how how we scope to `root` and `ifLet` the case inside root.